src/ResourceManagement/TrafficManager/TrafficManagerManagement/Properties/AssemblyInfo.cs

```diff
@@ -17,11 +17,11 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyTitle("Microsoft Azure Dns Management Library")]
-[assembly: AssemblyDescription("Provides Microsoft Azure Dns management functions for managing the Microsoft Azure Dns service.")]
+[assembly: AssemblyTitle("Microsoft Azure Traffic Manager Management Library")]
+[assembly: AssemblyDescription("Provides Microsoft Azure Traffic Manager management functions for managing the Microsoft Azure Traffic Manager service.")]
 
 [assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.9.1.0")]
+[assembly: AssemblyFileVersion("0.9.2.0")]
```

Comment:
you probably should back out the file version change, because your nuget.proj still uses older version <PackageVersion>0.9.1-preview</PackageVersion>.  Keep them in sync, till you add new functions and then bump up the version in nuget.proj, run "msbuild build.proj /t:build;package /p:scope=ResourceManagement\TrafficManager"